sys-apps/pick/files/pick-4.0.0-fix-build-for-clang16.patch

```diff
@@ -0,0 +1,67 @@
+Clang16 will not allow implicit function declarations and implicit integers etc.
+This patch overhauls the source code for modern C.
+
+Bug: https://bugs.gentoo.org/879771
+Upstream PR: https://github.com/mptre/pick/pull/316
+
+Signed-off-by: Pascal Jäger <pascal.jaeger@leimstift.de>
+
+--- a/configure
++++ b/configure
+@@ -66,6 +66,7 @@ check_dead() {
+ check_gnu_source() {
+ 	compile <<-EOF && return 1
+ 	#include <wchar.h>
```

Comment:
I thought there was a missing GNU_SOURCE here but actually no. I need to look at this properly.